src/gui/editors/SongEditor.cpp

```diff
@@ -1173,25 +1147,7 @@ void SongEditorWindow::lostFocus()
 }
 
 
-void SongEditorWindow::showZoomingSliderFloat()
-{
-	m_zoomStatus->moveGlobal(m_zoomingSlider, QPoint(m_zoomingSlider->width() + 2, -2));
-	m_zoomStatus->show();
-	updateZoomingSliderFloat(scaleToLinearZoom(m_editor->m_zoomingLogValue));
-}
-
-void SongEditorWindow::updateZoomingSliderFloat(int newVal)
-{
-	newVal = scaleToLogZoom(newVal);
 
-	m_editor->m_zoomingLogValue =newVal;
```

Comment:
As you may have seen by now, we don't store percentage values anywhere. The slider value is just some arbitrary linear scale that affects the precision of zoom. And for everything else we use pixelsPerBar(). Since a song doesn't have a "physical width" the use of 100% zoom is meaningless. It's not like a document that has a physical size on paper. Therefore I opted to remove the float, and let the user zoom to the desired level without knowing the exact value. This saves us some code and saves OCD people some anxiety over how hard it is to hit exactly 100% with the slider :stuck_out_tongue_winking_eye: